geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -563,14 +568,14 @@ vars:
           level: 3
         - name: printproxy_capabilities
           level: 3
-        - name: mapserverproxy
+        - name: mapserverproxy_wms_capabilitie
           display_name: mapserverproxy_wms
           params:
             SERVICE: WMS
             VERSION: 1.1.1
             REQUEST: GetCapabilities
           level: 3
-        - name: mapserverproxy
+        - name: mapserverproxy_wfs_capabilitie
```

Comment:
Is an "s" something missing here? "capabilitie" or "capabilities"?

Reply:
This was an error, reverted :-)